### PR TITLE
Issues/14 feat: rewards crud completed

### PR DESCRIPTION
**Pull request by JW-921**
#14 

影片展示:
https://github.com/user-attachments/assets/cf2805a7-739b-43b6-9b84-5519b6ca591f

### CODE REVIEW

File: rewards/migrations/0001_initial.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('description', models.TextField(blank=True)),
+                ('title', models.CharField(max_length=300)),
+                ('price', models.DecimalField(decimal_places=0, default=0, max_digits=10)),
+                ('ship_to', models.TextField()),
+                ('shipping_detail', models.TextField(blank=True)),
+                ('estimated_delivery', models.DateField(default=rewards.models.default_delivery_date)),
+                ('quantity', models.IntegerField(default=1)),
+                ('pledge_amount', models.DecimalField(decimal_places=0, default=10, max_digits=10)),
```
Comment (minirov1208) [inline rewards/migrations/0001_initial.py]: 贊助金額跟其他model重複了

File: rewards/forms.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+class RewardForm(forms.ModelForm):
+    class Meta:
+        model = Reward
+        fields = [
+            "title", 
+            "description",
+            "quantity",
+            "price", 
+            "pledge_amount",
```
Comment (minirov1208) [inline rewards/forms.py]: 這邊也要改

**Review by minirov1208 — APPROVED** [review]

**Review by IanYang1106 — APPROVED** [review]

**Comment by minirov1208** [comment]
有衝突

**Comment by JW-921** [comment]
已修復

**Review by Allen750411 — APPROVED** [review]

**Review by lioouzzz — APPROVED** [review]

**Review by viviennehsin — APPROVED** [review]

**Pull request merged**